README.md

```diff
@@ -67,16 +67,17 @@ The `polyfills.js` is packaged with this component, but **IT IS NOT NEEDED** to
 
 The `<auro-popover>` element should be used in situations where users may:
 
-* ...
-* ...
-* ...
+* Hover/blur over an element to display a tooltip-style graphic
```

Comment:
There is a lot of `tool-tip` and `popover` mentions. We need to get tighter on this. For example, in this phrase, I would state this once. 

`Hover/blur over an element to display a tooltip-style popover`